wscl-issues/proposed/return-from-error

```diff
@@ -0,0 +1,115 @@
+Issue:          RETURN-FROM-ERROR
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         proposed
+Edit History:   12-Sep-21, Version 1 by Tarn W. Burton.
+References:     RETURN-FROM
```

Comment:
Should probably reference `program-error` since that entry specifically mentions "… result from naming a go tag or a block tag that is not lexically apparent …"

Reply:
Sounds good